ecr-login/cli/docker-credential-ecr-login/main.go

```diff
@@ -14,13 +14,36 @@
 package main
 
 import (
+	"flag"
+	"fmt"
+	"os"
+
 	ecr "github.com/awslabs/amazon-ecr-credential-helper/ecr-login"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/api"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/config"
+	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/version"
 	log "github.com/cihub/seelog"
 	"github.com/docker/docker-credential-helpers/credentials"
 )
 
+var vrsn bool
+
+const BANNER = `amazon-ecr-credential-helper
+version: %s
+`
+
+func init() {
+	flag.BoolVar(&vrsn, "version", false, "print version and exit")
```

Comment:
Go's `flag` package doesn't properly support GNU-style long options.  If you want to use the `flag` package, I'd rather only support the `-v` short option instead of also having a long option that is non-standard.

Reply:
I only picked flag package since its in the standard packages. Would you prefer anything else, or is supporting -v enough? Do you envision the need for more flags or arguments?

Reply:
I'm fine supporting just `-v` right now.  I don't really expect to have more flags or arguments, since the typical workflow for this program is to be invoked indirectly by the Docker CLI.

Reply:
Fixed by 527b0da